combat-trainer.lic

```diff
@@ -2284,6 +2284,8 @@ class TrainerProcess
       'Barb Research Debilitation' => 'Debilitation',
       'Barb Research Utility' => 'Utility',
       'Barb Research Warding' => 'Warding',
+      'Landslide' => 'Warding',
+      'Avalanche' => 'Utility',
```

Comment:
❓  Thoughts on prefixing these terms to clarify what these abilities are? Similar to how `App` and `Barb Research` and `Khri` prefix a few other abilities?
```suggestion
      'Berserk Landslide' => 'Warding',
      'Berserk Avalanche' => 'Utility',
```

Reply:
Yeah, I think that's a good call.